ci-operator/config/openshift/cluster-api-provider-ibmcloud/openshift-cluster-api-provider-ibmcloud-main.yaml

```diff
@@ -0,0 +1,65 @@
+base_images:
+  base:
+    name: "4.9"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.16-openshift-4.8:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.16-openshift-4.8
+  openshift_release_golang-1.13:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+binary_build_commands: NO_DOCKER=1 make build
+build_root:
+  from_repository: true
+canonical_go_repository: sigs.k8s.io/cluster-api-provider-ibmcloud
```

Comment:
This should match the go module name, which is `github.com/openshift/cluster-api-provider-ibmcloud`